common/test/acceptance/tests/studio/test_studio_settings_certificates.py

```diff
@@ -196,3 +195,26 @@ def test_can_cancel_creation_of_certificate(self):
         certificate.course_title = "Title Override"
         certificate.click_cancel_edit_certificate()
         self.assertEqual(len(self.certificates_page.certificates), 0)
+
+    def test_line_breaks_in_signatory_title(self):
+        """
+        Scenario: Ensure that line breaks are properly reflected in certificate
+
+        Given I have a certificate with signatories
+        When I add signatory title with new line character
+        Then I see line break in certificate title
+        """
+        self.certificates_page.visit()
+        certificate = self.create_and_verify_certificate(
+            "Course Title Override",
+            0,
+            [self.make_signatory_data('Signatory title with new line character \n')]
+        )
```

Comment:
@saleem-latif this function will add \n character in each field of signatory while we are interested in title only as it is a textarea while others are <input>